dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -282,7 +282,7 @@ public Response toolsIdVersionsVersionIdGet(String id, String versionId, Securit
     @Override
     public boolean canExamine(User user, Entry entry) {
         return AuthenticatedResourceInterface.super.canExamine(user, entry)
```

Comment:
Just to make sure I understand correctly, line 284 essentially ends up calling `isOwner`, correct?
So this is checking that this workflow is either owned or the `permissionsInterface` gives permission.

Reply:
Yeah, that's correct